mbq/atomiq/__version__.py

```diff
@@ -4,4 +4,4 @@
 __license__ = 'Apache 2.0'
 __title__ = 'mbq.atomiq'
 __url__ = 'https://github.com/managedbyq/mbq.atomiq'
-__version__ = '0.0.12'
+__version__ = '0.1.0'
```

Comment:
This is a breaking change because it changes removes the argument `--celery-app` from the management command. Is this the right version bump?

Reply:
Or should it be `1.0.0`? or just `0.0.13`?

Reply:
According to semver any change starting with `0.0` should be interpreted as a breaking change